java-strings/src/main/java/com/baeldung/string/WordCount.java

```diff
@@ -0,0 +1,54 @@
+package com.baeldung.string;
+
+import java.util.StringTokenizer;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ *
+ * @author shwetaJ
+ */
+
+public class WordCount {
+	private static final Logger logger = LoggerFactory.getLogger(WordCount.class);
+
+	static int countUsingCharAt(String stringTocheck, char delimiter) {
+		int counter = 0;
+
+		char myChar[] = new char[stringTocheck.length()];
+		for (int i = 0; i < stringTocheck.length(); i++) {
+			myChar[i] = stringTocheck.charAt(i);
+			if (((i > 0) && (myChar[i] != delimiter) && (myChar[i - 1] == delimiter))
+					|| ((myChar[0] != delimiter) && (i == 0)))
+				counter++;
+		}
+		logger.info("Total no of words are: " + counter);
+
+		return counter;
+	}
+
+	/*
+	static int countUsingSpringStringUtils(String stringTocheck, String delimiter) {
+		return org.springframework.util.StringUtils.countOccurrencesOf(stringTocheck, delimiter) + 1;
+	}
+	*/
```

Comment:
Commented out code - will we remove it? or is it for the reader to uncomment? If so, mark it with a comment showing what they need to do to get it working.

Reply:
Yes, it is for the reader to uncomment, I 'll add a comment for reader here.